src/rpm/package.rs

```diff
@@ -196,8 +200,8 @@ impl Package {
                     if file_path.exists() || file_path.symlink_metadata().is_ok() {
                         fs::remove_file(&file_path)?;
                     }
-
-                    symlink(file.metadata.linkto, &file_path)?;
+                    entry_reader.finish()?;
```

Comment:
If `entry_reader.finish()` happens on every branch, is there a reason it can't be moved outside of the match block?